docs/use_cases/retrieval_augmented_generation_eval.md

```diff
@@ -0,0 +1,181 @@
+<!-- SEO: Retrieval augmented generation Evaluation - TODO Summary
+ -->
+
+# Evaluating Retrieval Augmented Generation
+
+
+## Understanding Retrieval-Augmented Generation
+
+<img src=/assets/use_cases/retrieval_augmented_generation_eval/rag_qdrant.jpg alt="Implementation of RAG using Qdrant as a vector database" data-size="100" />
+
+**Disclaimer**:
+The provided implementation serves as an example using  [Qdrant](https://qdrant.tech). Alternative Vector Databases are also available. To determine the most suitable Vector Database for your specific use case, please refer to the [Vector DB feature matrix](https://vdbs.superlinked.com/)
+
+**RAG** stands for **Retrieval Augmented Generation**, and it probably is the most useful application of large language models lately.
```

Comment:
maybe link to building blocks instead of explaining RAG in the article? @robertdhayanturner